src/components/careers/links.js

```diff
@@ -6,6 +6,6 @@ export const applicationHref = (typeformSlug) =>
 
 export const backstageLink = renderToString(
   <a href="https://github.com/backstage/backstage" target="_blank" rel="noopener noreferrer">
-    the open-source platform
+    Backstage
```

Comment:
Same comment regarding the `<Link />`. I realize you didn't add it.

Reply:
I think the <Link /> component would improve the experience so I will add it